src/osd/OSD.cc

```diff
@@ -9210,6 +9212,11 @@ void OSD::handle_conf_change(const struct md_config_t *conf,
     op_tracker.set_history_size_and_duration(cct->_conf->osd_op_history_size,
                                              cct->_conf->osd_op_history_duration);
   }
+  if (changed.count("osd_op_history_slow_op_size") ||
+      changed.count("osd_op_history_slow_op_threshold")) {
+    op_tracker.set_history_size_and_duration(cct->_conf->osd_op_history_slow_op_size,
+                                             cct->_conf->osd_op_history_slow_op_threshold);
```

Comment:
oops, should be `set_history_slow_op_size_and_threshold`.

The current historic ops are not the slowest ops, I think. The are only the most recent ops.